data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+			`	"What do I get in return?" you ask.`
+			`	"The ship, of course," Albert responds. "You've helped me so much, and you went to hell and back to get this black box decoded. We're going to spend a day or so studying the ship after we get it, making a blueprint and all that, but afterwards, it's all yours to keep! Anyway, I just can't wait to see what that ship even looks like. Give us some time to assemble a crew to help refurbish the ship, and in the meantime, get us a jump drive. Come back to us once you have it."`
+				accept
+	
+	to complete
+		never
+
+mission "The Emerald Sword 3"
+	name "The Emerald Sword"
+	description "Capture the Emerald Sword."
+	landing
+	source "Zug"
+	destination "Echoes of Tragedy"
+	cargo "Outfitting Equipment" 50
+	passengers 20
+	
+	to offer
+		has "The Emerald Sword 2: active"
+	
+	blocked `You arrive to make the preparations to capture the Emerald Sword, but the Archaeologists stop you. "Captain, we're going to need 50 tons of cargo to take our materials and the jump drive, and 20 additional crew to help, including us. I'd also advise that you save enough bunks to hire some additional crew. The Emerald Sword will need some people to man it."`
+	
+	on offer
+		require "Jump Drive"
+		conversation
+			`You have a jump drive in your possession. Would you like to give it to Albert's crew?`
+			choice
+				`    (No.)`
+					defer
+				`    (Yes.)`
+			`   You give Albert a call, and arrange to meet with him to plan the next step. He shows up at your ship, greets you, and starts loading in some equipment along with the jump drive you got. He has assembled a crew of around 20 people, including him and the three Hai archaeologists, now comfortable to let go of their disguise once they're back in your ship.`
+			`	"<first>, meet the people who will help us recover the Emerald Sword. I've just briefed them on the situation and what we plan to do. We're going to take some specialized tools with us, along with the Jump Drive, that will allow us to repair and outfit the ship in any world we can land on. I suggest you also hire some additional crew to man the ship once we capture it, since it's not going to pilot itself. If you're ready to go, Here are the coordinates to Zubenelhakrabi."`
+			`	You thank Albert, and make your preparations for the voyage into the unknown.`
+				accept
+	
+	on accept
+		fail "The Emerald Sword 2"
+		outfit "Jump Drive" -1
+		event "zubenelhakrabi discovered"
+	
+	on enter "Zubenelhakrabi"
+		conversation
+			`The trip to Zubenelhakrabi is completely normal and calm. Given everyone's trepidating expectations though, the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`
```

Comment:
I feel like this is unnecessarily loquacious :P

Reply:
Lol yeah, the word trepidatious might be a bit much, but I only changed it because trepidating isn't actually a word and I wanted to keep my change as close to your original idea as possible.

Reply:
Yeah, "trepidating" isn't a word, so something needs to change here.

Reply:
Oh, I was pretty sure it was. Is trepidatious a word? I'm sad i don't have a license to invent new words like Shakespeare :P

Reply:
>Is trepidatious a word?

Yes. 
https://www.dictionary.com/browse/trepidatious (Redirected to "trepidation." Scroll down to "other words from trepidation." See "trepidatious.")

Reply:
Why would you not just say "everyone's trepidation? Also, do not put a comma before and after though. Try this:
```suggestion
			`The trip to Zubenelhakrabi is completely uneventful. However, everyone's trepidation is such that the trip's uneventfulness only manages to keep everyone on edge. When you arrive, a simple scan reveals the system's star is unremarkable, nearing the end of its main sequence phase. Among the planets orbiting it, one is in the habitable zone and shows clear signs of life, covered in dark green spots under the clouds.`
```